bin/gimme-aws-creds.cmd

```diff
@@ -16,6 +16,11 @@ for /f "tokens=2 delims==" %%i in ('assoc .py') do (
         )
     )
 )
+for %%i in (cmd bat exe) do (
+    for %%j in (python.%%i) do (
+        call :SetPythonExe "%%~$PATH:j"
+    )
+)
```

Comment:
Basically copies lines 7-11 from above but using `python.%%i` instead of `python3.%%i`.

Reply:
This way it'll prefer `python3.[bat | cmd | exe]` from anywhere along the system path and the ftype results before falling back to a search for e.g., `python.[bat | cmd | exe]` on the system path.